templates/r10k.yaml.erb

```
@@ -33,3 +33,6 @@
 <% if !@forge_settings.empty? -%>
 <%= settings_to_yaml('forge', @forge_settings) %>
 <% end -%>
+<% if @proxy and @proxy != '' -%>
```

this can now be simplyfied:
```suggestion
 <% if @proxy -%>
```